operatorcourier/log_util.py

```diff
@@ -0,0 +1,67 @@
+import operatorcourier.log_dict_util as log_dict_util
+import logging.config
+from io import StringIO
+import json
+
+class log_util():
+    class log_util():
+        """
+        Wraps the Python logging module's logger object to ensure that all
+        logging is formatted into a json dictionary.
+        """
+        logger = None
+        
+        def __init__(self):
+            self._log_dict = dict(
+                    info=[],
+                    debug=[],
+                    warning=[],
+                    error=[],
+                    critical=[]
+                )
+            #logging.config.fileConfig("./config/logging.ini")
+            logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
```

Comment:
No, please, don't. You are setting logging outside main/cli in a module used by API again.
This will just ruin my preciously configured logging in my app.